pytest_gee/image_regression.py

```diff
@@ -23,6 +23,7 @@ def check(
         fullpath: Optional[os.PathLike] = None,
         scale: Optional[int] = 30,
         viz_params: Optional[dict] = None,
+        region: Optional[Union[ee.FeatureCollection, ee.Feature, ee.Geometry]] = None,
```

Comment:
I think GEE moved to Python 3.10 so we can finally use the piped hints: 

```suggestion
        region: ee.ComputedObject | None = None
```

Reply:
isn't `ee.ComputedObject` too broad?? `ee.Number` is a `ee.ComputedObject` 🤔

Reply:
yes it is but all the featurecollection that are getting out of an algorithm as well until they get realized with the getInfo()

Reply:
Hm. I am not convinced about this one. I think GEE is not that smart: https://code.earthengine.google.com/238eb13645a39810c4da7026b0ff6db4.
I also tested it in python:

![image](https://github.com/user-attachments/assets/2e0fca05-a985-4613-aaa8-5933abba2ed8)

Reply:
sad then forget about it